ClangSharpPInvokeGenerator/Properties/launchSettings.json

```diff
@@ -6,7 +6,7 @@
+      "commandLineArgs": "-m LLVM -p LLVM --namespace LLVMSharp --output \"./LLVMSharp/Generated/\" --config multi-file --libraryPath \"$(LibLLVMName)\" --include \"$(LLVMIncludePath)\" --file \"$(LLVMIncludePath)/llvm-c/Analysis.h\" --file \"$(LLVMIncludePath)/llvm-c/BitReader.h\" --file \"$(LLVMIncludePath)/llvm-c/BitWriter.h\" --file \"$(LLVMIncludePath)/llvm-c/Comdat.h\" --file \"$(LLVMIncludePath)/llvm-c/Core.h\" --file \"$(LLVMIncludePath)/llvm-c/DataTypes.h\" --file \"$(LLVMIncludePath)/llvm-c/DebugInfo.h\" --file \"$(LLVMIncludePath)/llvm-c/Disassembler.h\" --file \"$(LLVMIncludePath)/llvm-c/DisassemblerTypes.h\" --file \"$(LLVMIncludePath)/llvm-c/Error.h\" --file \"$(LLVMIncludePath)/llvm-c/ErrorHandling.h\" --file \"$(LLVMIncludePath)/llvm-c/ExecutionEngine.h\" --file \"$(LLVMIncludePath)/llvm-c/Initialization.h\" --file \"$(LLVMIncludePath)/llvm-c/IRReader.h\" --file \"$(LLVMIncludePath)/llvm-c/Linker.h\" --file \"$(LLVMIncludePath)/llvm-c/LinkTimeOptimizer.h\" --file \"$(LLVMIncludePath)/llvm-c/lto.h\" --file \"$(LLVMIncludePath)/llvm-c/Object.h\" --file \"$(LLVMIncludePath)/llvm-c/OptRemarks.h\" --file \"$(LLVMIncludePath)/llvm-c/OrcBindings.h\" --file \"$(LLVMIncludePath)/llvm-c/Support.h\" --file \"$(LLVMIncludePath)/llvm-c/Target.h\" --file \"$(LLVMIncludePath)/llvm-c/TargetMachine.h\" --file \"$(LLVMIncludePath)/llvm-c/Types.h\" --file \"$(LLVMIncludePath)/llvm-c/Transforms/AggressiveInstCombine.h\" --file \"$(LLVMIncludePath)/llvm-c/Transforms/Coroutines.h\" --file \"$(LLVMIncludePath)/llvm-c/Transforms/InstCombine.h\" --file \"$(LLVMIncludePath)/llvm-c/Transforms/IPO.h\" --file \"$(LLVMIncludePath)/llvm-c/Transforms/PassManagerBuilder.h\" --file \"$(LLVMIncludePath)/llvm-c/Transforms/Scalar.h\" --file \"$(LLVMIncludePath)/llvm-c/Transforms/Utils.h\" --file \"$(LLVMIncludePath)/llvm-c/Transforms/Vectorize.h\" -e LLVMDisasmInstruction -e LLVMGetBufferStart -e LLVMGetDiagInfoDescription -e LLVMGetDefaultTargetTriple -e LLVMCopyStringRepOfTargetData -e LLVMGetTargetMachineTriple -e LLVMGetTargetMachineCPU -e LLVMGetTargetMachineFeatureString -e LLVMPrintTypeToString -e LLVMCreateMessage -e LLVMDisposeMessage -e LLVMPrintModuleToString -e LLVMPrintValueToString -e LLVMOrcDisposeMangledSymbol -e LLVMTargetMachineEmitToFile -e LLVMInitializeCore -e LLVMAddInstructionCombiningPass -e LLVMAttributeIndex -e LLVMMetadataKind -r __AnonymousEnum_Core_16693=LLVMAttributeIndex -r __AnonymousEnum_DebugInfo_4207=LLVMMetadataKind"
```

Comment:
The change here is `-e LLVMAttributeIndex -e LLVMMetadataKind -r __AnonymousEnum_Core_16693=LLVMAttributeIndex -r __AnonymousEnum_DebugInfo_4207=LLVMMetadataKind"` and means that anonymous declarations can be handled.

There is still likely some minor fixups if the header file changes significantly, but it is a step in the right direction.